blog/2024-04-26-license-questions.mdx

```diff
@@ -0,0 +1,68 @@
+---
+slug: license-questions
+title: License Questions
+authors: [parker]
+tags: [panel, license]
+hide_table_of_contents: true
+---
+
+As some of you know, people have been asking us questions about when you need a license or not.
+
+I asked for people to get me questions they wanted clarifications on and here are some answers. 
+
+## modification questions
+
+1.  Q: What are the plans for the theming part of Pelican Panel and what will the front end look like?
+
+    A: Using [Filament themes](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme) placed in `/resources/css/filament` will work and don't need a license, 
+    and so long as no files that are included with the panel are modified in the process, you do not need a commercial license.
+
+:::info
+    A theme selector will eventually be implemented.
+:::
+:::warning
+    Themes are unable to change the layout of the panel, but plugins can.
+:::
+:::info definitions
+    Theme: The color and style of the panel. (Ex. button colors or fonts)  
+    Layout: The way the panel is organized. (Ex. where the cpu usage shows up.)
+    Panel Code: The files that the panel ship with.
+:::
+
+2.  Q: Will only styling be a thing, or is there also some sort of support for custom layouts and functionalities?
+
+    A: Custom layouts can be made with either plugins or by writing a custom frontend that uses the API to interact with the panel.
+
+3.  Q: Some users will probably use "public" themes for the panel (maybe smth like unix or so), if they are a public host, since they aren't the ones making the theme, do they need to pay the commercial license?
+
+    A: The answer for question 1 should answer this, but if a theme is public (as in open source) and does not modify the panel code if should be fine.
+
+:::warning
+    If it is closed source and modifies the panel code the host would have to get a license.
+:::
+
+
+## not license related
+    Still good questions
+
+1.  Q: What kind of SLA are you guys proposing for any commercial support plans?
+
+    A: Currently this depends on what level of support a customer would want. We are discussing options internally right now.
+
+2.  Q: Would it be possible for you guys to add a real data panel like splint or grafana or netdata for the server metrics?
+
+    A: Currently exporting server metrics to a service like Grafana is not planned but may be considered.
+:::info
+    As discussed on the discord server this may be better served by a monitoring service such as a host level netdata install.
+:::
+
+3.  Q: When Pelican goes public, will conversions be considered?
+
+    A: You can use your existing Pterodactyl configuration and drop in Pelican, it will update the database when installed.  
+:::warning
+    This would not be reversible at this time and it is heavily recommended that you back up your current configuration if you ever want to go back to Pterodactyl.   
```

Comment:
```suggestion
    We do not support reverting back to Pterodactyl. It isn't challenging, but we strongly recommend that you backup your files and database in case you want to.
```